DispatchTimer.swift

```diff
@@ -2,15 +2,15 @@
 import Foundation
 import UIKit
 
-public typealias Timer = DispatchTimer
```

Comment:
i can remove it all together but then we'll break the public API of the framework. i don't think it matters actually